librarian_packages/hera_librarian/hera_librarian/base_store.py

```diff
@@ -125,7 +125,7 @@ def copy_to_store(self, local_path, store_path):
             local_suffix = ''
 
         # flags: archive mode; keep partial transfers. Have SSH work in batch
-        # mode, use the arcfour256 cipher for speed, and turn off known hosts
+        # mode, use the aes128-ctr cipher for speed, and turn off known hosts
```

Comment:
Is aes128-ctr actually an unusually fast one? If not maybe just edit the commit and remove the cipher specification.

Reply:
Done.